docs/docs/cmd/spo/term/term-group-get.md

```diff
@@ -10,29 +10,95 @@ m365 spo term group get [options]
 
 ## Options
 
+`-u, --webUrl [webUrl]`
+: If specified, allows you to get a term group from the tenant term store as well as the sitecollection specific term store. Defaults to the tenant admin site.
+
 `-i, --id [id]`
-: ID of the term group to retrieve. Specify `name` or `id` but not both
+: ID of the term group to retrieve. Specify `name` or `id` but not both.
 
 `-n, --name [name]`
-: Name of the term group to retrieve. Specify `name` or `id` but not both
+: Name of the term group to retrieve. Specify `name` or `id` but not both.
 
 --8<-- "docs/cmd/_global.md"
 
 ## Remarks
 
 !!! important
-    To use this command you have to have permissions to access the tenant admin site.
+    To use this command without the `--webUrl` option you have to have permissions to access the tenant admin site.
+
+When using the `--webUrl` option you can connect to the term store with limited permissions, and do not need the SharePoint Adminstrator role. You need be a site visitor or more. It allows you to get a term group from the tenant term store as well as a term group from the sitecollection term store.
 
 ## Examples
 
-Get information about a taxonomy term group using its ID
+Get information about a taxonomy term group using its ID.
 
 ```sh
 m365 spo term group get --id 0e8f395e-ff58-4d45-9ff7-e331ab728beb
 ```
 
-Get information about a taxonomy term group using its name
+Get information about a taxonomy term group using its name.
 
 ```sh
 m365 spo term group get --name PnPTermSets
 ```
+
+Get information about a taxonomy term group using its ID from the specified sitecollection.
```

Comment:
Get information about a taxonomy term group from the specified sitecollection using its ID.